include/exadg/grid/moving_mesh_function.h

```diff
@@ -81,10 +81,10 @@ class MovingMeshFunction : public MovingMeshBase<dim, Number>
     FE_Nothing<dim> dummy_fe;
```

Comment:
I'm wondering if the `initialize` function is still needed. It seems that `update` is only directed here without doing its own work, and I can't find uses of `MovingMeshFunction::initialize()` elsewhere. So I would simply move this whole function into `update()`.

Reply:
@kronbichler I made the function private. I would currently like to keep the function itself as it makes the code slightly more structured and provides additional documentation, e.g., that the function is interpreted as a displacement.

Reply:
I agree with the documentation aspect, so it is fine for me to merge (even though I do not see a real advantage in terms of structure because it merely adds an indirection).

Reply:
I am probably just a bit pedantic about code readability ;). To give an example, the line that sets the time for the function would somehow get lost when merging the two function into one. In the current form, the procedural design is more clear in my opinion. This is something of interest to the user. The implementation details are then part of a private member function.